nav2_bringup/launch/nav2_bringup_launch.py

```diff
@@ -52,9 +54,8 @@ def generate_launch_description():
         launch_ros.actions.Node(
             package='dwb_controller',
             node_executable='dwb_controller',
-            node_name='FollowPathNode',
             output='screen',
-            parameters=[{ 'prune_plan': False }, {'debug_trajectory_details': True }, { 'use_sim_time': use_sim_time }]),
+            parameters=[params_file]),
```

Comment:
When you rebase, you'll see that the map server has a different interface. You can pass the map YAML filename as a **parameter** instead of an **argument**.  I think the map server should also get the use_sim_time parameter since it sets the map load time using node_->now().